### DISCUSSION TITLE
can't searh the repository `tldr`

### DISCUSSION THREAD

**Discussion by tjx666** [community/community#74393] [Repositories]
### Select Topic Area

Product Feedback

### Body

Maybe this disscuss is off topic, but I can't find a suitable top to dicuss it.

<img width="1495" alt="image" src="https://github.com/community/community/assets/41773861/1b3e706d-e4a8-4fcd-948c-f8326299a874">

[tldr](https://github.com/tldr-pages/tldr) is a popular project, but can't be searched.

<img width="1489" alt="image" src="https://github.com/community/community/assets/41773861/0902a997-970a-4ffb-bc89-1bb48b4e5e82">

**Comment by look** [comment]
I could not reproduce this. Can you let us know if it's still affecting you?

**Comment by tjx666** [comment]
close because I can't reproduce it now.
